<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LRwindow</class>
 <widget class="QMainWindow" name="LRwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="I_Table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>421</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>State</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ItemSet</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="parse_Table">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>651</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>700</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Courier New&quot;;
color: qconicalgradient(cx:0.045, cy:0.528409, angle:180.3, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
    </property>
    <property name="text">
     <string>Query is Accepted!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Consolas&quot;;
text-decoration: underline;
color: rgb(225, 225, 255);</string>
    </property>
    <property name="text">
     <string>Parse Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Consolas&quot;;
text-decoration: underline;
color: rgb(225, 225, 255);</string>
    </property>
    <property name="text">
     <string>Cannonical Form</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bgImage/graphics.webp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>I_Table</zorder>
   <zorder>parse_Table</zorder>
   <zorder>ResultLabel</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bgimage.qrc"/>
 </resources>
 <connections/>
</ui>
